tools/symbols_map_generator/main.py

```diff
@@ -75,6 +75,35 @@ def create_node_symbol_name(node: clang.cindex.Cursor) -> str:
         return node.spelling
 
 
+def is_virtual_class_or_struct(node: clang.cindex.Cursor):
```

Comment:
This name seems dubious.: It doesn't refer to virtual [base] classes, but to classes that have (or inherit) virtual methods.

Straw man: `has_virtual_member_functions()`

Reply:
Or even `has_vtable()` - since that's what the calling code wants to know

Reply:
`has_vtable` works for me

Reply:
Done :+1: